workflow-fingerprint-compare/README.md

```diff
@@ -0,0 +1,72 @@
+# fingerprintCompare
+
+Workflow to generate jaccard matrices for all projects.
+
+## Overview
+
+## Dependencies
+
+* [perl 5.30](https://www.perl.org/)
+
+
+## Usage
+
+### Cromwell
+```
+java -jar cromwell.jar run fingerprintCompare.wdl --inputs inputs.json
+```
+
+### Inputs
+
+#### Required workflow parameters:
+Parameter|Value|Description
+---|---|---
+`finFile`|File|Input .fin file containing list of .fin files
```

Comment:
perhaps this option is better as finList, as it point to a file that list the fin files and not a fin file itself